cmd/manager/main.go

```diff
@@ -63,27 +62,28 @@ func main() {
 		log.Fatal(err)
 	}
 
-	operatorStatus := status.New(cfg, mgr, namespace, os.Getenv("RELEASE_VERSION"))
-	operatorStatus.SetProgressing(fmt.Sprintf("Version %s of the operator is being deployed", operatorStatus.GetVersion()))
-
 	log.Print("Registering Components.")
 
 	catalogsourceconfig.InitializeStaticSyncer(mgr.GetClient(), initialWait)
 	registrySyncer := operatorsource.NewRegistrySyncer(mgr.GetClient(), initialWait, resyncInterval, updateNotificationSendWait, catalogsourceconfig.Syncer, catalogsourceconfig.Syncer)
 
+	// monitorStopCh is used to send a signal to stop reporting cluteroperator status
+	monitorStopCh := make(chan struct{})
+	monitorDoneCh := status.StartReporting(cfg, mgr, namespace, os.Getenv("RELEASE_VERSION"), monitorStopCh)
```

Comment:
Nit: Given you commented about `monitorStopCh` you should comment about `monitorDoneCh` does.